### DISCUSSION TITLE
Is it possible to block specific browser extensions for specific websites?

### DISCUSSION THREAD

**Discussion by Dean-Corso** [uBlockOrigin/uAssets#21705] [Q&A] [answered]
Hi guys,

just have a little question and would like to know whether it's anyhow doable to block specific Extension / Add-ons with uBlockOrigin? So I see they using other protocols...
```
chrome-extension://abc123...
moz-extension://abc123...
```
...and trying to block the extension addresses (value xy) does not work. But could it be possible that uBO can or could do that too? I was looking for an method to block / disable any extension on certain domains when they gets loaded in browser tab just to prevent to disable them manually each time you know. Problem in Chrome is I can not blacklist websites for any extension only whitelist. Otherwise I found a Chrome extension called "[Extension Manager](https://chromewebstore.google.com/detail/extension-manager/gjldcdngmdknpinoemndlidpcabkggco)" which can work to enable / disable extension on websites I want but then it does do it for all at once and not tab specific where I have loaded the websites. So the extension is ok so far +/- but normally I would not like to use a extra extension for this task and that's why I'm asking whether uBO could do that possibly anyhow.

**Comment by ItsProfessional** [comment, marked as answer]
Pretty sure that uBO can't do this.
